src/tabs/borderless.scss

```diff
@@ -49,4 +44,18 @@
       }
     }
   }
+
+  &.iui-large {
+    > li > button {
+      height: $iui-baseline * 6;
+    }
+  }
+
+  ~ .iui-tabs-content {
```

Comment:
Does content area really need to have border for other types? I feel like it would add too much visual noise

Reply:
We talked about this in MS Teams chat, seems like we are in agreement on making the top border an option as some times they may want it and other times may not.